.ado/windows-vs-pr.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           configuration: $(BuildConfiguration)
           publishRunAttachments: true
           collectDumpOn: onAbortOnly
-          vsTestVersion: 15.0
+          vsTestVersion: latest
         condition: and(eq(variables.BuildPlatform, 'x64'), eq(variables.BuildConfiguration, 'Debug'))
```

Comment:
>x64 [](start = 52, length = 3)

I have seen that you have added cdecl to unit tests. Does it mean that they now compile for x86? I have disabled them before in the solution file. If they compile, then can we remove this restriction then?

Reply:
Yes, with cdecl you can build the unit tests in x86. The google tests libs are only cdecl, which is the default for x64, which is why they were building.

Reply:
@jonthysell Please reconsider. Compatibility issues are introduced every now and then. We should use the specific version number.

Reply:
The sooner we find issues the better. Otherwise we end up with "catch-up" PRs like this, just to get on current tools. When we snap off stable branches, we can lock the CI setup in that branch, but master is bleeding edge now.